src/Neon/gSPNeon.cpp

```diff
@@ -2,7 +2,7 @@
 #include <math.h>
 #include <algorithm>
 #include <assert.h>
-#include "N64.h"
+#include "../N64.h"
```

Comment:
Have  you tried to force Xcode to look at source folder for headers?
May be CMAKE_CURRENT_SOURCE_DIR can help?
For example, replace
include_directories( . inc )
by
include_directories( ${CMAKE_CURRENT_SOURCE_DIR} inc )

It looks strange that it can't find headers located in folders contained in includes path.

Reply:
It was finding headers in other folders like Rice and causing conflicts. I had to use 'Always search user paths' combined with 'User header maps' to 'off' which is a setting I had to look up but it makes Xcode not go around looking all over the project dir for headers of the same name. I'll revert these changes in a short bit.